### PR TITLE
Update hello-hybrid-key-whfb-settings-policy.md

### PR DESCRIPTION
**Pull request by pamcdo**
The ", and" is part of the dialog box and should be bolded as well. Not bolding it has led customers to believe there are two separate options instead of one.

### CODE REVIEW

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-settings-policy.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Sign-in a domain controller or management workstations with _Domain Admin_ equiv
 7. Expand **Windows Settings**, **Security Settings**, and click **Public Key Policies**.
 8. In the details pane, right-click **Certificate Services Client � Auto-Enrollment** and select **Properties**.
 9. Select **Enabled** from the **Configuration Model** list.
-10. Select the **Renew expired certificates**, **update pending certificates**, and **remove revoked certificates** check box.
+10. Select the **Renew expired certificates**, **update pending certificates**, **and remove revoked certificates** check box.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-settings-policy.md:58]:
```suggestion
10. Select the **Renew expired certificates, update pending certificates, and remove revoked certificates** check box.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by e0i** [comment]
@pamcdo The PR is good to go after the copy editing suggestions have been applied. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

**Pull request merged**